Lib/test/test_ctypes/test_dllist.py

```diff
@@ -0,0 +1,72 @@
+import os
+import sys
+import test.support
+import unittest
+from ctypes import CDLL
+from ctypes.util import dllist
+
+
+WINDOWS = os.name == 'nt'
+APPLE = sys.platform in {"darwin", "ios", "tvos", "watchos"}
+AIX = sys.platform.startswith("aix")
+
+if WINDOWS:
+    SYSTEM_LIBRARY = 'KERNEL32.DLL'
+elif APPLE:
+    SYSTEM_LIBRARY = 'libSystem.B.dylib'
+elif AIX:
+    SYSTEM_LIBRARY = None
+else:
+    SYSTEM_LIBRARY = 'libc.so'
```

Comment:
Is there a performance benefit to this? It might suffice to just change this to:

```suggestion
SYSTEM_LIBRARY = find_library("c")
```

Reply:
I think it is a bit more self-documenting to list out a library we know we expect on each platform. Some sort of conditional is required in order to handle unsupported platforms (AIX), or Windows, since:

> On Windows, [find_library()](https://docs.python.org/3/library/ctypes.html#ctypes.util.find_library) searches along the system search path, and returns the full pathname, but since there is no predefined naming scheme a call like find_library("c") will fail and return None.